WooCommerce/Classes/ViewRelated/Products/Edit Product/Bundled Products/Bundled Product List/BundledProductsListViewModel.swift

```diff
@@ -37,14 +41,34 @@ final class BundledProductsListViewModel {
 
 // MARK: Initializers
 extension BundledProductsListViewModel {
-    convenience init(bundledProducts: [Yosemite.ProductBundleItem]) {
-        let viewModels = bundledProducts.map {
-            BundledProduct(id: $0.bundledItemID, title: $0.title, stockStatus: $0.stockStatus.description)
+    convenience init(siteID: Int64, bundledProducts: [Yosemite.ProductBundleItem], storageManager: StorageManagerType = ServiceLocator.storageManager) {
+        let products = BundledProductsListViewModel.fetchProducts(for: siteID, including: bundledProducts.map { $0.productID }, storageManager: storageManager)
```

Comment:
Confidence check. Can we assume that we will always have the products already synced?

Reply:
Ah, good thought. It would be possible to view this list but not have _all_ products synced yet. (For example, if only the first page of products is synced but one of the bundled products isn't on the first page.)

I'll go ahead and merge this for now, and then confirm how we want to handle the loading UI if we need to sync the products from remote to get their image URLs. (I think it'd be better to show the information we have immediately, even if we need to load the images afterward, but I'll check that and handle the syncing in another PR.)